packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -6,11 +6,12 @@ import isA from '@hapi/joi';
 import * as Sentry from '@sentry/node';
 import { Account } from 'fxa-shared/db/models/auth';
 import { ACTIVE_SUBSCRIPTION_STATUSES } from 'fxa-shared/subscriptions/stripe';
+import { isArray } from 'lodash';
```

Comment:
VS Code trying to be helpful here?

Reply:
Looks like it, surprised that remained when it wasn't used.